tests/Pest.php

```
@@ -21,6 +27,49 @@
+    foreach (vitoArchFiles($directory) as $file) {
+        if (preg_match('/^(?:final |abstract |readonly )*class /m', $file->getContents()) !== 1) {
+            continue;
+        }
+
+        $class = Str::of($file->getRealPath())
+            ->after(app_path().DIRECTORY_SEPARATOR)
+            ->replace([DIRECTORY_SEPARATOR, '.php'], ['\\', ''])
+            ->prepend('App\\')
+            ->toString();
+
+        if (! class_exists($class) || (new ReflectionClass($class))->isAbstract()) {
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '63', column '44'). (undefined)

(MissingImport)